### ISSUE TITLE
Frothy tear film + Tear film anomaly

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10870]
Preferred term label:
Foamy tear film

Synonyms
Frothy tear film, Tear foam, Tear bubbles

Definition (free text, please give PubMed ID)
A condition characterized by the presence of foam or bubbles within the tear film, often associated with meibomian gland dysfunction or ocular surface diseases. [PMID:35508212](https://pubmed.ncbi.nlm.nih.gov/35508212/)

Parent term (use hpo.jax.org/app)
a parent term such as "Tear film anomaly" is also missing.

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Meibomian gland dysfunction, Blepharitis

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01  -- adding as a subclass of Lacrimation abnormality for now, since there is not other tear film abnormality and there would be little use in a parent with just one child!
Thanks for the many excellent suggestions!

Adding new term: 
- ID: HP:6001229
- label: Foamy tear film
- parents: [Lacrimation abnormality (HP:0000632)](https://hpo.jax.org/app/browse/term/HP:0000632)
- pmid: PMID:35508212
- def: A condition characterized by the presence of foam or bubbles within the tear film, often associated with meibomian gland dysfunction or ocular surface diseases.
- comment: This feature may be observed in conditions such as Meibomian gland dysfunction and blepharitis.
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10870
- synonyms: Frothy tear film(NONE)|Tear bubbles(NONE)|Tear foam(NONE)
